libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/utils/RouteUtils.kt

```diff
@@ -0,0 +1,98 @@
+package com.mapbox.services.android.navigation.v5.utils
+
+import com.mapbox.geojson.Point
+import com.mapbox.services.android.navigation.v5.routeprogress.RouteProgress
+import com.mapbox.services.android.navigation.v5.routeprogress.RouteProgressState
+import java.util.ArrayList
+import java.util.Arrays
+
+class RouteUtils {
```

Comment:
Noting here that we should move `RouteUtils` to `internal`.

Reply:
Didn't move to internal since the class is still public and the keyword internal is not used at the moment. I think we can leave it there for now just to avoid confusion.